k8s/helm/templates/garbage-collection-manager.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: batch/v1
```

Comment:
File naming: I am not sure but somehow I didn't expect a cronjob here. There are two sets of workload here,
- deployment and
- cronjob

I suggest to change either naming to have an idea that here is more than the superficial eye meets, like manager-deployment and garbage-collection-manager-cronjob or sth. like it

Reply:
Oh yes. I will use garbage-collection-manager-cronjob